src/Engine/ProtoScript/Runners/LiveRunner.cs

```diff
@@ -1172,7 +1172,7 @@ private void InitCore()
             runnerCore.Options.IncludeDirectories = configuration.SearchDirectories.ToList();
             foreach (var item in configuration.PassThroughConfiguration)
             {
-                runtimeCore.DSExecutable.RuntimeData.Configurations[item.Key] = item.Value;
+                runnerCore.Configurations[item.Key] = item.Value;
             }
```

Comment:
Could you stick this into a separate PR as discussed as it is related to fixing a separate crash.